src/vue/pages/Signup.vue

```diff
@@ -92,6 +112,9 @@ export default {
       loadKyc: vuexTypes.LOAD_KYC,
       loadKvEntriesAccountRoleIds: vuexTypes.LOAD_KV_ENTRIES_ACCOUNT_ROLE_IDS,
     }),
+    showAccessButton (value) {
```

Comment:
* what is "access button"? you don't use this term anywhere else
* what is "sending"? you're disabling the button, not the action itself
* why method that "shows" disables or enables something instead of changing it's visibility?
* why method that "shows" works in two directions? from the method name I assume that it can only "show", from the method body I assume it can also "hide"